upload-api/types.ts

```diff
@@ -264,4 +264,19 @@ declare module '@serverless-stack/node/config' {
   }
 }
 
```

Comment:
here on top should come type for STRIPE_SECRET_KEY right?

Reply:
ah good call! done!